lambda-code/submission/src/lib/fileUpload.ts

```diff
@@ -35,10 +35,21 @@ interface FileInput {
 }
 
 export function findAttachedFileReferencesInSubmissionResponses(
-  responses: Record<string, unknown>
+  responses: Record<string, unknown>,
+  submissionId: string
 ): FileReference[] {
   const fileInputs = extractFileInputs(responses);
-  return fileInputs.map((f) => ({ id: f.id, name: f.name }));
+
+  // Log and transform
+  return fileInputs.map((f) => {
+    console.log(
+      JSON.stringify({
+        level: "info",
+        msg: `File input detected for submission ${submissionId}: fileID=${f.id}, fileSize=${f.size} bytes.`,
```

Comment:
Any reason why you added the submission and file identifier?
I was originally thinking about a simple file size log tailored specifically for the new metric.